src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -107,37 +120,33 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
         )
       );
 
-      //todo tools functionality needs to be added to toolbar
-      testIf(theme !== 'refresh-toolbar')(
-        'focuses tools panel closed button when it is opened using keyboard and caused split panel to change position',
+      test(
+        'focuses tools panel closed button when it is opened via keyboard and caused split panel to change position',
         setupTest(
           async page => {
-            await page.setWindowSize({ width: 1000, height: 800 });
+            // Mobile nav is closed on page load
+            mobile && (await page.click(wrapper.findNavigationToggle().toSelector()));
+            await page.setWindowSize({ width: 1100, height: 800 });
```

Comment:
So, in this file we have some desktop/mobile dual tests. But this test sets the fixed size here, which makes it run twice as "mobile" and "desktop" with the same actual screen size.

No questions for the test flow, but we can be more frugal and extract it outside  that `[true, false].forEach(mobile =>` loop

Reply:
That was my original approach, but because it did fail on mobile otherwise (because it has this different navigation behavior), I felt it was still providing value.